Makefile

```diff
@@ -26,6 +26,9 @@ test:
 test_v1a2:
 	./05_test_v1a2.sh
 
+test_v1a3:
+	./05_test_v1a2.sh
```

Comment:
Script name should not be version specific if used like this.

Reply:
Is it a typo or are we using v1a2 test scripts for v1a3 also?

Reply:
This is just to keep the CI jobs green. Once the v1a3 target exists, we'll change it